code/planning/src/behavior_agent/behaviors/stop_mark_service_utils.py

```
@@ -0,0 +1,104 @@
+def create_stop_marks_proxy() -> rospy.ServiceProxy:
+    service = rospy.ServiceProxy("/paf/hero/mapping/update_stop_marks", UpdateStopMarks)
+    service.wait_for_service()
+    return service
+
+
+def update_stop_marks(
+    proxy: rospy.ServiceProxy,
+    id: str,
+    reason: str,
+    is_global: bool = False,
+    marks: List[Union[StopMark, Shape2D, shapely.Polygon]] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)